sw/device/tests/rv_core_ibex_address_translation_test.S

```diff
@@ -0,0 +1,106 @@
+/**
+ * Returns the integer 5 always.
+ *
+ * @return 5
+ */
+  .globl return_five
+  .type return_five, @function
+return_five:
```

Comment:
Forgot to remove this unused toy function. Will remove when I get back.

Reply:
I'm assuming you're talking about this whole assembly file? Or do you need the other functions?